test/functionalTest/cases/2496_mq_metadata_not_exist/mq_metadata_not_exist.test

```diff
@@ -0,0 +1,139 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This is funny... it should be part of https://github.com/telefonicaid/fiware-orion/pull/2512 but it seems that it wasn't added in that moment.

So this PR now includes it. Problem fixed. NTC.